nextcord/gateway.py

```diff
@@ -782,24 +804,32 @@ class DiscordVoiceWebSocket:
         _max_heartbeat_timeout: float
         thread_id: int
 
-    def __init__(self, socket, loop, *, hook=None):
-        self.ws = socket
-        self.loop = loop
-        self._keep_alive: Any = None
-        self._close_code = None
-        self.secret_key = None
-        self._hook = hook or getattr(self, "_hook", None) or getattr(self, "_default_hook")
-
-    async def _default_hook(self, *args):
+    def __init__(
+        self,
+        socket: DiscordClientWebSocketResponse,
+        loop: asyncio.AbstractEventLoop,
+        *,
+        hook: Optional[Callable] = None,
+    ):
+        self.ws: DiscordClientWebSocketResponse = socket
+        self.loop: asyncio.AbstractEventLoop = loop
+        self._keep_alive: Optional[VoiceKeepAliveHandler] = None
+        self._close_code: Optional[int] = None
+        self.secret_key: Optional[str] = None
+        self._hook: Callable = (
```

Comment:
Be more specific with this type hint. I.e. use the brackets after the type hint itself and specify parameters (if you're confused on what I mean, look at the [docs](https://docs.python.org/3/library/typing.html#callable) for more info)